root/static/scripts/tests/GuessCase.js

```diff
@@ -188,8 +188,6 @@ test('Work', function (t) {
     ];
 
     $.each(tests, function (idx, test) {
-        var gc = window.gc;
-
         gc.CFG_UC_ROMANNUMERALS = test.roman;
```

Comment:
https://github.com/mwiencek/musicbrainz-server/blob/node-tests/root/static/scripts/guess-case/MB/GuessCase/Main.js#L135

It's still a global, as the (redundant) removed line showed, so I removed the `var gc = window.gc;` line because there's no need to shadow it as a local variable.